web/client/components/data/query/QueryToolbar.jsx

```diff
@@ -104,7 +105,8 @@ const QueryToolbar = React.createClass({
             sortOptions: this.props.sortOptions,
             hits: this.props.hits
         };
-        this.props.actions.onToggleDrawer();
```

Comment:
same here, there should be an thunk that opens the bottom table and closes the left menu.
Better if it happens when the search result arrives.